ls1intum/ArTEMiS
# Feature/add details panel to exercise list

### Checklist

### Motivation and Context
This is a missing feature of the student dashboard.

### Description
I added a information panel on the right about the current course in the exercise list of the new student dashboard

### Steps for Testing

1. Log in to ArTEMiS
2. Navigate to Course Overview
3. Select a course
4. There should now be course details panels
5. check if the numbers are correct
6. check if the upcoming exercises are clickable

### Screenshots
![image](https://user-images.githubusercontent.com/10998002/53949350-292e9180-40ca-11e9-847c-c316f363167c.png)
